README.md

```diff
@@ -161,15 +195,34 @@ You can work with KotlinDL interactively in Jupyter Notebook with the Kotlin ker
 For more details on installing Jupyter Notebook and adding the Kotlin kernel, check out the [Quick Start Guide](docs/quick_start_guide.md).
 
 ### Working with KotlinDL in Android projects
+
 KotlinDL supports an inference of ONNX models on the Android platform.
 To use KotlinDL in your Android project, add the following dependency to your build.gradle file:
+```groovy
+// build.gradle
+implementation 'org.jetbrains.kotlinx:kotlin-deeplearning-onnx:[KOTLIN-DL-VERSION]'
+```
 ```kotlin
-dependencies {
-    implementation ("org.jetbrains.kotlinx:kotlin-deeplearning-onnx:[KOTLIN-DL-VERSION]")
-}
+// build.gradle.kts
+implementation ("org.jetbrains.kotlinx:kotlin-deeplearning-onnx:[KOTLIN-DL-VERSION]")
 ```
 For more details, please refer to the [Quick Start Guide](docs/quick_start_guide.md#working-with-kotlin-dl-in-android-studio).
 
+## KotlinDL, ONNX Runtime, Android, and JDK versions
+
+This table shows the mapping between KotlinDL, TensorFlow, ONNX Runtime, Compile SDK for Android and minimum supported Java versions.
+
+| KotlinDL Version | ONNX Runtime Version | Minimum Java Version | TensorFlow Version | Android: Compile SDK Version |
+|------------------|----------------------|----------------------|--------------------|------------------------------|
+| 0.1.0            |                      | 8                    | 1.15               |                              |
+| 0.1.1            |                      | 8                    | 1.15               |                              |
+| 0.2.0            |                      | 8                    | 1.15               |                              |
+| 0.3.0            | 1.8.1                | 8                    | 1.15               |                              |
+| 0.4.0            | 1.11.0               | 8                    | 1.15               |                              |
+| 0.5.0            | 1.12.1               | 11                   | 1.15               | 31                           |
+| 0.5.1            | 1.12.1               | 11                   | 1.15               | 31                           |
+| 0.6.0-SNAPSHOT   | 1.12.1               | 11                   | 1.15               | 31                           |
```

Comment:
Latest version is `0.6.0-alpha-1`.

Reply:
We can use wildcards here, like `0.6.*` or `0.6+` (to avoid needing to add new lines for each new version), we can also use this to shorten the table and combine several versions together, like `0.1.0-0.2.0`.